### ISSUE TITLE
LLVMPipe: Sometimes there is no image when booting GNOME in a VM

### ISSUE THREAD

**Issue by joebonrichie** [getsolus/packages#1107]
### Summary

When booting the GNOME edition in an VM with LLVMPipe graphics enabled an image doesn't always appear on boot. An reset is generally required to fix it.

### Steps to reproduce

1. Load up the GNOME ISO in virt-manager

### Expected result

A graphical session to always appear

### Actual result

Sometimes no graphical appears

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

GNOME

### System details

# System Details Report
---

## Report details
- **Date generated:**                              2023-12-23 17:23:52

## Hardware Information:
- **Hardware Model:**                              QEMU Standard PC _Q35 + ICH9, 2009_
- **Memory:**                                      4.0 GiB
- **Processor:**                                   AMD Ryzen™ 7 3700X × 2
- **Graphics:**                                    Software Rendering
- **Disk Capacity:**                               26.8 GB

## Software Information:
- **Firmware Version:**                            rel-1.16.2-0-gea1b7a073390-prebuilt.qemu.org
- **OS Name:**                                     Solus 4.5 Resilience
- **OS Build:**                                    (null)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               45.2
- **Windowing System:**                            X11
- **Kernel Version:**                              Linux 6.6.8-268.current

### Other comments

I vaguely remember this being an issue at least since the 4.4 ISO

This doesn't seem to happen with other editions so could be a GNOME specific bug or a bug in llvmpipe/mesalib

**Comment by WetGeek** [comment]
I don't use virt-manager, because it makes it too hard to access my NAS.  But on VirtualBox, there are two different video adapters for SVGA when choosing settings for a VM.  VBoxSVGA works fine for all the other Solus editions, but if I use that for creating a GNOME VM, the result is as you said - no image appears, just a blank screen.  If I use the VMware emulator, the VMSVGA setting, GNOME works*.  I don't know if this will help you, but you might check to see whether virt-manager also offers a choice when it comes to the video adapter.

* For some value of "works."  I don't have much appreciation for GNOME.

**Comment by TraceyC77** [comment]
I tried booting the Gnome 4.5 RC5 ISO in virt-manager and VirtualBox (UEFI). In virt-manager I got a black background with an X cursor. 

The VB VM wouldn't boot at all, with any Graphics Controller, so I couldn't test with that.
I got an error (which is a VBox problem, not a Gnome problem)
```
VirtualBox can't operate in VMX root mode. Please disable the KVM kernel extension, recompile your kernel and reboot (VERR_VMX_IN_VMX_ROOT_MODE).
```
TL;DR, VirtualBox does not play nicely on a system with virt-manager also installed. 

I was able to boot to the graphical installer normally using a live USB on a Lenovo Flex.

**Comment by WetGeek** [comment]
I find this very curious.  I have no problems creating a VirtualBox VM so long as I use the VMSVGA video adapter (the Vmware one) instead of the VboxSVGA one that works fine with the rest of the DEs.  I'm not disputing that what you're saying is true, just pointing out that my experience is different.  I've needed to use the VMSVGA adapter for GNOME VMs for as long as I can remember, but it's never failed to boot for me.  The VboxSVGA adapter always fails with GNOME.

**Comment by TraceyC77** [comment]
Joey verified this issue also exists in Fedora and Serpent, so it's an upstream thing somewhere
